Notification templates (#190)
OHS-Hosting-Infrastructure/infrastructure: docs/how-we-work/notification-templates/acknowledgment-message.md

This template is for acknowledgement messages that don't require immediate investigation and fixing.

The message should be sent within 2 hours of receipt during [core business hours](https://github.com/OHS-Hosting-Infrastructure/infrastructure/tree/main/docs/how-we-work#working-hours)
If message was received outside code business hours, it should be sent next business day.

If an issue was reported as a ticket, please use ticket to acknowledge receipt using mentions in a comment.

If an issue was reported as an email, send an email response and ask for a ticket.

## When to Use

> If an issue is P1 or P2 according to docs/how-we-work/support-process.md

Reviewer: ```suggestion
If an issue is P1 or P2 according to [our support process doc definition](../support-process.md)
```
